egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -87,6 +102,24 @@ def PrintConfig(file_name, config_lines):
     f.write("\n".join(config_lines['component-nodes']))
     f.close()
 
+def ParseCnnString(cnn_string):
+    cnn_array = []
+    cnn_list={}
+    for cl in range(0, len(cnn_string)):
+      split1 = cnn_string[cl].split(" "); 
+      if len(split1) != 11:
+        sys.exit("invalid --cnn-layer input, must contain 11 arguments for one CNN layer: "
```

Comment:
Actually i am using --cnn-layer to let user to choose whether to use CNN or not, if this option is not used, arg.cnn_layer is None then LDA is used instead of CNN